example_text_completion.py

```diff
@@ -61,4 +61,4 @@ def main(
 
 
 if __name__ == "__main__":
-    fire.Fire(main)
+    fire.Fire(main)
```

Comment:
what is the change here (and in the other example script)?

Reply:
Fixed the `No newline at end of file` issue in this and the other example script by enabling the VS Code setting to automatically add a newline 😅